static/src/javascripts/projects/common/modules/experiments/tests/participation-low-fric-fashion.js

```diff
@@ -0,0 +1,74 @@
+define([
+    'common/utils/config',
+    'common/utils/mediator',
+    'Promise'
+], function (
+    config,
+    mediator,
+    Promise
+) {
+
+    var omniture;
+
+    /**
+     * The omniture module depends on common/modules/experiments/ab, so trying to
+     * require omniture directly inside an AB test gives you a circular dependency.
+     *
+     * This is a workaround to load omniture without making it a dependency of
+     * this module, which is required by an AB test.
+     */
+    function getOmniture() {
```

Comment:
This could totally be a util but considering we're removing omniture I don't want to spread it.